tuplex/adapters/cpython/include/PythonHelpers.h

```diff
@@ -319,9 +319,10 @@ namespace python {
     /*!
      * get corresponding tuplex type for python object
      * @param o
+     * @param autoUpcast false by default
      * @return
      */
-    extern python::Type mapPythonClassToTuplexType(PyObject *o);
+    extern python::Type mapPythonClassToTuplexType(PyObject *o, bool autoUpcast);
```

Comment:
where is default = false set? Would recommend to set the default param where the function is declared